README.md

```diff
@@ -1,2 +1,132 @@
 # calendar-date
-Immutable object to represent a calendar date with zero dependencies.
+
+A calendar date is a date without time information, e.g. "2020-01-01".
+This library provides an immutable object to represent and work with a calendar date.
+
+[![License](https://img.shields.io/badge/license-MIT-blue.svg)](LICENSE)
+
+## Why another date library?
+
+There are a lot of date libraries in the javascript world, but there is currently no good support for representing a calendar date.
+Most SQL Databases like MySQL and PostgreSQL have a datatype for representing a date without time information.
+
+Using the build-in date object for a calendar date poses a number of disadvantages:
+- Overhead by including time and timezone information in the representation.
+- Error prone when working with different timezones and converting from and to Strings.
+  For example `new Date(2020, 0, 1)` and `new Date('2020-01-01')` return date objects with a difference of one hour.
+  If you would call `toISOString()` on the first date object it would return 2019-12-31T23:00:00.000Z.
+- If you are using typescript you can not directly tell from the type information if the object is representing a timestamp or a calendar date.
+- It's harder to compare two date objects if you don't know if the time and timezone information are the same.
+
+## Features
+
+- CalendarDate and CalendarDateRange
+- Immutable API
+- Written completely in typescript
+- No external dependencies
+- Tiny package size
+
+## Getting Started
+
+Install with yarn or npm:
+
+```sh
+yarn add calendar-date
+```
+
+or
+
+```sh
+npm install calendar-date
+```
+
+## Documentation
+
+### CalendarDate
+
+You can construct a CalendarDate from an ISOString, a unix timestamp in seconds or the year, month and date values.
+The following constructor calls return the same calendar date.
+
+```typescript
+new CalendarDate('2020-01-01');
+new CalendarDate(1577833200);
+new CalendarDate(2020, 0, 1);
+```
+
+The year, month, day and unix timestamp can be accessed as read-only properties on the object.
+
+To compare to CalendarDate objects you can use the `equals` function or `>=` and `<=` operators.
+The comparison works based on the unix timestamp.
+
+```typescript
+const date1 = new CalendarDate('2020-01-01');
+const date2 = new CalendarDate('2020-01-01');
+const date3 = new CalendarDate('2020-02-02');
+
+date1.equals(date2) // true
+date1.equals(date3) // false
+date1 >= date2      // true
+date1 >= date3      // false
+date1 <= date3      // true
+```
+Utility Methods:
+#### addMonths, addDays
+Returns a new CalendarDate with the specified amount of months or days added.
```

Comment:
```suggestion
Returns a new CalendarDate with specified amount of months or days added.
```

Reply:
It hurts to say this, but Patrick's version is correct :D